core/storage/app_feedback_report/gae_models.py

```diff
@@ -470,34 +484,31 @@ class AppFeedbackReportStatsModel(base_models.BaseModel):
     # to the creation date of the reports aggregated in this model.
     stats_tracking_date = datastore_services.DateProperty(
         required=True, indexed=True)
+    # The total number of reports submitted on this date.
+    total_reports_submitted = datastore_services.IntegerProperty(
+        required=True, indexed=True)
     # JSON struct that maps the daily statistics for this ticket on the date
-    # specified in stats_tracking_date. The JSON will contain two keys:
-    # daily_param_stats and daily_total_reports_submitted.
-    # The daily_param_stats will map each param_name (defined below
-    # by a domain const ALLOWED_STATS_PARAM_NAMES) to a dictionary of all the
-    # possible param_values for that parameter and the number of reports
+    # specified in stats_tracking_date. The JSON will map each param_name
+    # (defined by a domain const ALLOWED_STATS_PARAM_NAMES) to a dictionary of
+    # all the possible param_values for that parameter and the number of reports
     # submitted on that day that satisfy that param value":
     #
-    #   daily_param_stats : { param_name1 : { param_value1 : report_count1,
+    #   daily_param_stats = { param_name1 : { param_value1 : report_count1,
```

Comment:
Drop `daily_param_stats =` and put a sentence/clause before that introduces the format, e.g. `...that satisfy the param value, similar to e.g.:`

Reply:
Done